lte/gateway/python/magma/pipelined/ebpf/ebpf_manager.py

```diff
@@ -260,10 +260,11 @@ def print_ul_map(self):
             egress_dev_name = self._get_if_name(egress_dev_index)
             dst_mac = self._unpack_mac_addr(v.mac_dst)
             src_mac = self._unpack_mac_addr(v.mac_src)
+            bytes = v.bytes
 
             print(
```

Comment:
shouldn't these be `logging` ?

Reply:
maybe, but for now I use it only via direct script invocation for debugging so lets keep it print